deeposlandia/static/app.js

```diff
@@ -0,0 +1,19 @@
+// Jitenshea
+
+// PREFIX should be defined in the settings.js file
+var API_URL = PREFIX + '/api';
+
+// Build the URL with a BASE_URL/<city> suffix based from a DOM element with the
+// "city" dataset attribute, i.e. 'data-city'.
+function cityurl(dom_id) {
+  return API_URL + "/" + document.getElementById(dom_id).dataset.city;
+};
+
+// get the date before today in YYYY-MM-DD string format
+function getYesterday() {
```

Comment:
Is this function mandatory? Maybe you don't neet it.

Reply:
Yes, indeed. Actually the whole `app.js` is useless for `deeposlandia`. I remove it.